clar2wasm/src/words/equal.rs

```diff
@@ -1000,32 +1000,29 @@ mod tests {
 
     #[test]
     fn index_of_list_not_present() {
-        crosscheck(
-            "(index-of? (list 1 2 3 4 5 6 7) 9)",
-            Ok(Some(Value::none())),
-        );
+        crosscheck("(index-of (list 1 2 3 4 5 6 7) 9)", Ok(Some(Value::none())));
```

Comment:
here and below: should we also keep the `index-of?` variant of the test? To ensure it works for clarity version > 1; eg duplicating the mod tests, or something smarter

also for `clar2wasm/tests/wasm-generation/equal.rs`

Reply:
`index_of` and `index_of?` are aliases pointing to the same code. Anyway, I've duplicated some tests to make sure `index_of?` works in Clarity versions > 1. too